src/dom/dom.js

```diff
@@ -2227,11 +2246,12 @@ p5.prototype.createCapture = function(...args) {
     if (arg === p5.prototype.VIDEO) useAudio = false;
     else if (arg === p5.prototype.AUDIO) useVideo = false;
     else if (typeof arg === 'object') {
-      constraints = arg;
-      flipped = constraints.flipped || false;
-    }
-    else if (typeof arg === 'boolean') {
-      flipped = arg;
+      // Check if the argument is an object with a 'flipped' property
+      if (arg.flipped !== undefined) {
+        flipped = arg.flipped;
+      } else {
+        constraints = arg;
```

Comment:
Right now I think this means we either can provide `flipped`, or we can provide constraints on the capture size. I think we want to support both at once, but just with the `flipped` arg removed. Maybe something like:
```js
if (arg.flipped !== undefined) {
  flipped = arg.flipped;
}
constraints = { ...arg };
delete constraints.flipped;
```